docs/release_notes/v1.7.3.md

```diff
@@ -0,0 +1,15 @@
+# Dapr 1.7.3
+
+### Fixes incorrect actor state-store configuration behavior
+
+#### Problem
+
+Omitting a state store configuration will cause the actor subsystem to fail to initialize. This is the correct behavior for services providing actors but this not for services acting as consumers / clients of actors that other services provide. A service acting purely as a consumer does not need state storage and should initiazlize correctly in order to invoke actors elsewhere. 
```

Comment:
"state storage" -> "actor state storage" (both occurrences)

"but this not for" -> "but not for"

"A service acting purely as a consumer" -> client instead of consumer